src/stp/new_plays/GenericPass.cpp

```diff
@@ -67,17 +67,17 @@ Dealer::FlagMap GenericPass::decideRoleFlags() const noexcept {
     Dealer::DealerFlag closeToBallFlag(DealerFlagTitle::CLOSE_TO_BALL, DealerFlagPriority::HIGH_PRIORITY);
     Dealer::DealerFlag receiverFlag(DealerFlagTitle::WITH_WORKING_DRIBBLER, DealerFlagPriority::REQUIRED);
 
-    flagMap.insert({"keeper", {keeperFlag}});
+//    flagMap.insert({"keeper", {keeperFlag}});
     flagMap.insert({"passer", {closeToBallFlag}});
-    flagMap.insert({"receiver", {receiverFlag}});
-    flagMap.insert({"halt_0", {not_important}});
-    flagMap.insert({"halt_1", {not_important}});
-    flagMap.insert({"halt_2", {not_important}});
-    flagMap.insert({"halt_3", {not_important}});
-    flagMap.insert({"halt_4", {not_important}});
-    flagMap.insert({"halt_5", {not_important}});
-    flagMap.insert({"halt_6", {not_important}});
-    flagMap.insert({"halt_7", {not_important}});
+//    flagMap.insert({"receiver", {receiverFlag}});
+//    flagMap.insert({"halt_0", {not_important}});
+//    flagMap.insert({"halt_1", {not_important}});
+//    flagMap.insert({"halt_2", {not_important}});
+//    flagMap.insert({"halt_3", {not_important}});
+//    flagMap.insert({"halt_4", {not_important}});
+//    flagMap.insert({"halt_5", {not_important}});
+//    flagMap.insert({"halt_6", {not_important}});
+//    flagMap.insert({"halt_7", {not_important}});
```

Comment:
Haico is a little bit disappointed if he sees this kind of stuff :disappointed:, never out comment dead code but remove it instead...